python/tvm/contrib/hexagon/build.py

```diff
@@ -360,10 +361,6 @@ def _copy_binaries(self):
     def _run_server_script(self):
         """Setup the ADB connection and execute the server script."""
 
-        # Removed pre-defined forward/reverse rules
-        subprocess.check_call(self._adb_device_sub_cmd + ["forward", "--remove-all"])
```

Comment:
Do we want to clean up port forwards that have been opened for more than N days?  The `.stop_server()` method may not be called if a segfault occurs, and that may leave ports open unnecessarily.